quickstart_guides/ruby.md

```diff
@@ -2,3 +2,84 @@ address: support.aptible.com/quickstart/ruby
 
 ### Ruby Quickstart
 
+This guide assumes you have:   
+- An Aptible account,  
+- An SSH key associated with your Aptible user account, and  
+- The Aptible command line tool installed
+
+#### 1. Provision Your App  
+Tell the Aptible API that you want to provision an application. Until you push code and trigger a build, Aptible uses this as a placeholder.
+
+##### Command line
+Use the `apps:create` command: `aptible apps:create HANDLE`
+
+For example: 
+```
+aptible apps:create my-example-app
+```
+
+If you have multiple accounts, you will need to specify `--account <account_handle>`. Each account handle is listed next to the account name in the Aptible Dashboard.
+
+For example: 
+```
+aptible apps:create example-app --account myco-development
+```
+
+##### Aptible Dashboard  
+From the [Aptible dashboard](https://dashboard.aptible.com/apps), pick an account to provision the app into, select "Create App," and choose a handle.
+
+#### 2. Add a Dockerfile
+Aptible uses Docker to build and run applications. To tell Aptible how to build and run your specifica application, you must include a Dockerfile in your repository.
+
+A few guidelines:  
+1. Name the file "Dockerfile" - one word, capital 'D', no extension  
+2. Place it in the root of your repository  
+3. Be sure to commit your Dockerfile to version control  
+
+Here is a sample Dockerfile for a Ruby on Rails app:
+```
+FROM quay.io/aptible/ruby:ruby-2.0.0
+
+RUN apt-get update && apt-get -y install libpq-dev nodejs
+ 
+ADD . /opt/rails
+WORKDIR /opt/rails
+RUN bundle install --without development test
+RUN cp config/database.yml.example config/database.yml
+RUN bundle exec rake assets:precompile
+ 
+ENV PORT 3000
+EXPOSE 3000
+ 
+CMD bundle exec rails s -p $PORT
+```
+
+#### 3. Configure a Git Remote
+```
+git remote add <remote_name> git@beta.aptible.com:<app_handle>.git
```

Comment:
The `<remote_name>` here should be `aptible` (both because that's how it's referenced afterwards, and because it's necessary for proper functioning of aptible-cli).